extras/util/strings.go

```diff
@@ -10,3 +15,43 @@ func StringSplitArg(stringToSplit, separator string) []interface{} {
 	}
 	return splitInterface
 }
+
+func Normalize(s string) string {
+	c := cases.Fold()
+	return c.String(norm.NFD.String(s))
+}
+
+func ReverseBytes(s []byte) {
+	for i, j := 0, len(s)-1; i < j; i, j = i+1, j-1 {
+		s[i], s[j] = s[j], s[i]
+	}
+}
+
+// convert txid to txHash
+func ToHash(txid string) []byte {
+	t, err := hex.DecodeString(txid)
+	if err != nil {
+		return nil
+	}
+
+	// reverse the bytes. thanks, Satoshi 😒
+	for i, j := 0, len(t)-1; i < j; i, j = i+1, j-1 {
+		t[i], t[j] = t[j], t[i]
+	}
+
+	return t
+}
+
+// convert txHash to txid
+func FromHash(txHash []byte) string {
```

Comment:
maybe call this TxHashToTxid()? since these are in lbry.go now, its a generic location and it's not clear what they're for

Reply:
Yeah makes sense. I'm also taking a look at this function and it's only used in a print statement, nothing really relies on it, but if I understand the semantic of copy, I'm not sure it works as intended, it's copying the hash to t, then reversing the bytes on the hash, but then decoding and returning t, I think it wants to reverse the bytes on t?

Reply:
yea that's weird. it should reverse t. the whole point is that the reverse does it in-place so we copy to avoid messing with the original

Reply:
here's what should really happen: ReverseBytes should stop reversing in place. it should return  a copy. or alternatively,  it should be called ReverseInPlace or something. otherwise its very surprising.